tests/pytorch_tests/model_tests/feature_models/qat_test.py

```diff
@@ -165,20 +170,24 @@ def run_test(self):
                      quantization_info=quantization_info)
 
     def compare(self, ptq_model, qat_ready_model, qat_finalized_model, input_x=None, quantization_info=None):
-        all_trainable_quantizers = get_all_subclasses(BasePytorchQATTrainableQuantizer)
+        all_qat_weight_quantizers = get_all_subclasses(BasePytorchQATWeightTrainableQuantizer)
+        all_qat_activation_quantizers = get_all_subclasses(BasePytorchActivationTrainableQuantizer)
         # check relevant layers are wrapped and correct quantizers were chosen
         for _, layer in qat_ready_model.named_children():
             if isinstance(layer, PytorchActivationQuantizationHolder):
-                q = [_q for _q in all_trainable_quantizers if _q.identifier == self.training_method
+                q = [_q for _q in all_qat_activation_quantizers if _q.identifier == self.training_method
                      and _q.quantization_target == QuantizationTarget.Activation
                      and self.activation_quantization_method in _q.quantization_method]
                 self.unit_test.assertTrue(len(q) == 1)
                 self.unit_test.assertTrue(isinstance(layer.activation_holder_quantizer, q[0]))
+                self.unit_test.assertFalse(layer.activation_holder_quantizer.freeze_quant_params)
```

Comment:
maybe add a comment to the assertion, something like "in QAT we expect all learnable parameters to be trained"

Reply:
fixes in #1197